GRDB/Record/EncodableRecord+Encodable.swift

```diff
@@ -149,9 +150,9 @@ private class RecordEncoder<Record: EncodableRecord>: Encoder {
             } catch is JSONRequiredError {
                 // Encode to JSON
                 try autoreleasepool {
-
+                    
```

Comment:
This looks like a missing editor setting to trim trailing white space?

Reply:
GRDB does not trim trailing white spaces.

Reply:
OK, cool. Just makes the diff bigger is all. Your project, no problem. 👍